src/main/pages/HomePageOnet.java

```diff
@@ -55,30 +52,35 @@ public String acceptCookies2() {
         return getLogoText();
     }
 
-    public void getArticleLinksOpenThemInDifferentTabsAndSaveLinkstoFile() throws IOException {
+    public void getArticleLinksOpenInTabsAndSavetoFile() throws IOException {
         System.out.println("Saving article links to the list");
-        List<WebElement> phraseSearch2 = driver.findElements(By.partialLinkText("ska"));
-        WebElement q;
-        String k;
+        /**
+         Change "ska" to whatever text you want to search it on www.onet.pl site
+         */
+        List<WebElement> searchByPhrase = driver.findElements(By.partialLinkText("ska"));
+        WebElement phraseSearch;
+        String singleArticle;
```

Comment:
Redundant variables, delete

Reply:
That's true, variables are redundant and already deleted (change will appear in the next commit).